docs/install/uninstallEverest.md

```diff
@@ -8,98 +8,227 @@ You can run the commands below to remove all Everest resources including:
+    customresourcedefinition.apiextensions.k8s.io "vmauths.operator.victoriametrics.com" deleted
+    customresourcedefinition.apiextensions.k8s.io "vmprobes.operator.victoriametrics.com" deleted
+    customresourcedefinition.apiextensions.k8s.io "vmpodscrapes.operator.victoriametrics.com" deleted
+    customresourcedefinition.apiextensions.k8s.io "vmsingles.operator.victoriametrics.com" deleted
+    customresourcedefinition.apiextensions.k8s.io "vmstaticscrapes.operator.victoriametrics.com" deleted
+    customresourcedefinition.apiextensions.k8s.io "vmservicescrapes.operator.victoriametrics.com" deleted
+    customresourcedefinition.apiextensions.k8s.io "vmalertmanagerconfigs.operator.victoriametrics.com" deleted
+    customresourcedefinition.apiextensions.k8s.io "vmalertmanagers.operator.victoriametrics.com" deleted
+    customresourcedefinition.apiextensions.k8s.io "vmalerts.operator.victoriametrics.com" deleted
+    customresourcedefinition.apiextensions.k8s.io "vmagents.operator.victoriametrics.com" deleted
+    customresourcedefinition.apiextensions.k8s.io "vmclusters.operator.victoriametrics.com" deleted
+    customresourcedefinition.apiextensions.k8s.io "vmusers.operator.victoriametrics.com" deleted
+    customresourcedefinition.apiextensions.k8s.io "databaseclusterbackups.everest.percona.com" deleted
+    customresourcedefinition.apiextensions.k8s.io "databaseclusters.everest.percona.com" deleted
+    customresourcedefinition.apiextensions.k8s.io "databaseengines.everest.percona.com" deleted
+    customresourcedefinition.apiextensions.k8s.io "backupstorages.everest.percona.com" deleted
+    customresourcedefinition.apiextensions.k8s.io "databaseclusterrestores.everest.percona.com" deleted
+    customresourcedefinition.apiextensions.k8s.io "monitoringconfigs.everest.percona.com" deleted
+    customresourcedefinition.apiextensions.k8s.io "perconaxtradbclusterbackups.pxc.percona.com" deleted
+    customresourcedefinition.apiextensions.k8s.io "perconaxtradbclusterrestores.pxc.percona.com" deleted
+    customresourcedefinition.apiextensions.k8s.io "perconaxtradbclusters.pxc.percona.com" deleted
+    customresourcedefinition.apiextensions.k8s.io "perconaservermongodbbackups.psmdb.percona.com" deleted
+    customresourcedefinition.apiextensions.k8s.io "perconaservermongodbrestores.psmdb.percona.com" deleted
+    customresourcedefinition.apiextensions.k8s.io "perconaservermongodbs.psmdb.percona.com" deleted
+    customresourcedefinition.apiextensions.k8s.io "perconapgbackups.pgv2.percona.com" deleted
+    customresourcedefinition.apiextensions.k8s.io "perconapgrestores.pgv2.percona.com" deleted
+    customresourcedefinition.apiextensions.k8s.io "perconapgclusters.pgv2.percona.com" deleted
+    customresourcedefinition.apiextensions.k8s.io "postgresclusters.postgres-operator.crunchydata.com" deleted
     ```
 
-5. Remove ALL PVCs:
+#### Remove CRD for PostreSQL operator
```

Comment:
```suggestion
#### Remove CRDs of Percona's PostgreSQL operator but keep Crunchy Data's PostgreSQL operator CRDs
```

Reply:
This is a very long title. I do not think it is a good idea to include this.